Lib/typing.py

```diff
@@ -2065,6 +2064,50 @@ class Other(Leaf):  # Error reported by type checker
     return f
 
 
+class NewType:
```

Comment:
What was the reason to move this definition?

Reply:
`_type_check()` was used before defining NewType.